Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -508,6 +501,29 @@ kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o jsonpa
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o jsonpath='{.data.SecretKey}' | base64 --decode
 ```
 
+## Enable TLS
```

Comment:
Added this doc section to help users with TLS, advertiseEndpoint, and certificate management.